code/__DEFINES/subsystems.dm

```diff
@@ -47,7 +47,7 @@
 
 #define INIT_ORDER_GARBAGE 19
 #define INIT_ORDER_DBCORE 18
-#define INIT_ORDER_BLACKBOX 17
+#define INIT_ORDER_LOGGING 17
```

Comment:
Reused this one since it wasn't used anywhere

Reply:
This will be used by the blackbox SS when I make it. Also if this is logging OOC stuff I recommend making it one of the first things to init if it needs to log OOC pre-round.

Reply:
I've changed it back and made it go before the DBCORE